browser/main/Main.js

```diff
@@ -144,6 +152,28 @@ class Main extends React.Component {
     }
   }
 
+  handleFullScreenButton (e) {
+    this.setState({ fullScreen: !this.state.fullScreen }, () => {
+      const noteDetail = document.querySelector('.NoteDetail')
+      const mainBody = document.querySelector('#main-body')
+      const sliderRight = document.querySelector('#slider-right')
+      const sliderLeft = document.querySelector('#slider-left')
+      if (this.state.fullScreen) {
+        this.state.noteDetailWidth = noteDetail.style.left
+        this.state.mainBodyWidth = mainBody.style.left
+        noteDetail.style.left = '0px'
+        mainBody.style.left = '0px'
+        sliderRight.style.display = 'none'
+        sliderLeft.style.display = 'none'
+      } else {
+        noteDetail.style.left = this.state.noteDetailWidth
+        mainBody.style.left = this.state.mainBodyWidth
+        sliderRight.style.display = 'block'
+        sliderLeft.style.display = 'block'
+      }
+    })
+  }
```

Comment:
`sliderRight` and `sliderLeft` are important because if you are in the editor mode, the slider bars to move the lists appear through the editor. You would be able to drag those sliders which is not good. I need to turn those off or display none them while you are in fullscreen. 

I do like how you separated the code into 3 functions. I do not like the name leftLists, it is not descriptive. I do not quite know what to call them. If they have different names we should use them? I just called them lists because I don't know what else to call them.